pom.xml

```diff
@@ -1041,7 +1041,7 @@
         <version>2.2</version>
         <configuration>
           <sourceDirectory>
-            ${basedir}/src/main/resources/com/puppycrawl/tools/checkstyle/grammars
+            src/main/resources/com/puppycrawl/tools/checkstyle/grammar
```

Comment:
Please restore basedir usage, if it if not required, it should be done in separate commit or PR.